src/pages/sign-up/SignUp.tsx

```diff
@@ -45,19 +45,26 @@ const SignUp = () => {
   })
   const formSubmitHandler: SubmitHandler<IFormInputs> = (data: IFormInputs) => {
     {
+      let accountSvcError=false
       console.log(data)
       data.password = hashSync(data.password, bcrypt.genSaltSync(10))
       CreateAccountService.getCreateAccount(data)
         .then((response) => {
           if (response.data) {
             dispatch(autoPopulateEmailAction(data.email))
+            console.log(response)
+            navigate('/signin')
+          }
+          else{
+            navigate('/signin',{state:{accountSvcError:accountSvcError}})
           }
-          //TODO
           console.log(response)
         })
         .catch((error) => {
-          //TODO
           console.log(error)
+          accountSvcError=true;
+          console.log(error)
+          navigate('/signin',{state:{accountSvcError:accountSvcError}})
```

Comment:
it will show Account already exists error message on any kind of backend errors. not only on create an account with existing email error